client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-utils.js

```diff
@@ -0,0 +1,8 @@
+export function hasStripeKeyPairForMode( method ) {
```

Comment:
Would be great to have some unit tests around this at some point.

Reply:
Will do so in next PR